CONTRIBUTING.md

```diff
@@ -65,9 +65,7 @@ local KCP server and install the KubeStellar components and run an
 example.  As a contributor you will want a different setup flow,
 including `git clone` of this repo instead of fetching and unpacking a
 release archive.  The same example usage should work for you, and
-there is a larger example in
-https://docs.kubestellar.io/docs/coding-milestones/poc2023q1/example1/
-.
+there is a larger example in https://docs.kubestellar.io/release-v0.2.2/Coding%20Milestones/PoC2023q1/example1/.
```

Comment:
Ugh.  An explicit version number.  This ties us in knots: in the commit of the latest release, it can not refer to its own version --- unless an earlier commit refers to the upcoming version that does not exist yet.

Since this is aimed at contributors, I think replacing the specific release reference with `main`.

Reply:
Alternatively we could use a relative reference that is always right but not rendered ideally, with instructions for getting the best rendering.  Something like the following.

> there is a larger example in [example1.md](docs/content/Coding%20Milestones/PoC2023q1/example1.md) (see [the website](https://kubestellar.io) for the best rendering).

Reply:
I was going to ask the same. 'main' works

Reply:
I removed the specific release version with `main`